man/mysql.1

```diff
@@ -1677,15 +1678,15 @@ variable to
 .IP \(bu 2.3
 .\}
 Create
-\&.mysql_history
+\&.mariadb_history
 as a symbolic link to
 /dev/null:
 .sp
 .if n \{\
 .RS 4
 .\}
 .nf
-shell> \fBln \-s /dev/null $HOME/\&.mysql_history\fR
+shell> \fBln \-s /dev/null $HOME/\&.mariadb_history\fR
 .fi
```

Comment:
might be worth adding a **NOTE** section with, like, "it'll fallback to MYSQL_HISTFILE and .mysql_history if mariadb* counterparts don't exist"

Reply:
This was covered in the body already.